src/data/network/pci_subclasses.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Nice, this is much more readable!
And so the problem gets exposed: where does the magic mapping come from, and what do the strings mean? We should add a comment about that.
Is it a PCI standard? A kernel thing? A hwinfo thing?